assets/src/edit-story/elements/text/edit.js

```diff
@@ -64,6 +66,7 @@ const Wrapper = styled.div`
     border: 1px solid ${({ theme }) => theme.colors.mg.v1}70;
     pointer-events: none;
   }
+  background-color: ${({ theme }) => theme.colors.whiteout};
```

Comment:
Could we move this line up to [line 57](https://github.com/google/web-stories-wp/pull/2554/files#diff-5f4f624867fd96363e2603afb736f301R57) in stead? It's a bit weird here at the bottom...